app/src/main/java/com/hover/stax/transactions/TransactionHistoryAdapter.kt

```diff
@@ -0,0 +1,89 @@
+package com.hover.stax.transactions
+
+import com.hover.stax.utils.DateUtils.humanFriendlyDate
+import com.hover.sdk.actions.HoverAction
+import androidx.recyclerview.widget.RecyclerView
+import com.hover.stax.transactions.TransactionHistoryAdapter.HistoryViewHolder
+import android.view.ViewGroup
+import android.view.LayoutInflater
+import android.view.View
+import androidx.core.text.HtmlCompat
+import androidx.recyclerview.widget.DiffUtil
+import androidx.recyclerview.widget.ListAdapter
+import com.hover.sdk.transactions.Transaction
+import com.hover.stax.channels.Channel
+import com.hover.stax.databinding.TransactionListItemBinding
+import java.util.Locale
+
+class TransactionHistoryAdapter(private var listOfTransactionHistory: List<TransactionHistory>,
+                                private val selectListener: SelectListener)
+	: ListAdapter<TransactionHistory, HistoryViewHolder>(diffUtil) {
+
+	fun updateData(listOfHistory:  List<TransactionHistory>) {
+		this.listOfTransactionHistory = listOfHistory
+		notifyDataSetChanged()
+	}
+
+	override fun onCreateViewHolder(parent: ViewGroup, viewType: Int): HistoryViewHolder {
+		val binding =
+			TransactionListItemBinding.inflate(LayoutInflater.from(parent.context), parent, false)
+		return HistoryViewHolder(binding)
+	}
+
+	override fun onBindViewHolder(holder: HistoryViewHolder, position: Int) {
+		val history = listOfTransactionHistory[position]
```

Comment:
To get one item, use the `getItem(adapterPosition)` method.

Reply:
Thanks for the tip